src/atomate2/openmm/jobs/core.py

```diff
@@ -0,0 +1,250 @@
+
+        Parameters
+        ----------
+        input_set : .OpenMMSet
+            A pymatgen structure object.
+        prev_dir : str or Path or None
+            A previous OpenMM simulation directory to link files from.
+
+        Returns
+        -------
+        Job
+            A OpenMM job containing one npt run.
+        """
+
+        platform = Platform.getPlatformByName("OpenCL")
+        sim = input_set.get_simulation(
+            platform=platform,
+            platformProperties={"DeviceIndex": str(0)},
+        )
+
+        context = sim.context
+        system = context.getSystem()
+        assert (
+            system.usesPeriodicBoundaryConditions()
+        ), "system must use periodic boundary conditions for pressure equilibration."
+        barostat_force_index = system.addForce(
+            MonteCarloBarostat(self.pressure * atmosphere, self.temperature * kelvin, 10)
+        )
+        context.reinitialize(preserveState=True)
+        sim.step(self.steps)
+        system.removeForce(barostat_force_index)
+        context.reinitialize(preserveState=True)
+
+        return input_set
+
+
+@dataclass
+class NVTMaker(BaseOpenmmMaker):
+    steps: int = 1000000
+    name: str = "npt simulation"
+    temperature: float = 298
+    pressure: float = 1
+    frequency: int = 10
+
+    def make(self, input_set: OpenMMSet, prev_dir: Optional[Union[str, Path]] = None):
+        """
+        Equilibrate the temperature of a simulation in the NPT ensemble.
+
+        Adds and then removes a openmm.MonteCarlo Barostat to shift the system
+        into the NPT ensemble.
+
+        Parameters
+        ----------
+        input_set : .OpenMMSet
+            A pymatgen structure object.
+        prev_dir : str or Path or None
+            A previous OpenMM simulation directory to link files from.
+
+        Returns
+        -------
+        Job
+            A OpenMM job containing one npt run.
+        """
+        platform = Platform.getPlatformByName("OpenCL")
+        sim = input_set.get_simulation(
+            platform=platform,
+            platformProperties={"DeviceIndex": str(0)},
+        )
+
+        context = sim.context
+        system = context.getSystem()
+        assert (
+            system.usesPeriodicBoundaryConditions()
+        ), "system must use periodic boundary conditions for pressure equilibration."
+        thermostat_force_index = system.addForce(
+            AndersenThermostat(self.temperature * kelvin, self.frequency * pico * second)
+        )
+        context.reinitialize(preserveState=True)
+        sim.step(self.steps)
+        system.removeForce(thermostat_force_index)
+        context.reinitialize(preserveState=True)
+
+        return input_set
+
+
+@dataclass
+class AnnealMaker(BaseOpenmmMaker):
+
+    steps: Tuple[int, int, int] = (500000, 1000000, 500000)
+    name: str = "npt simulation"
+    temperatures: Tuple[int, int, int] = (298, 400, 298)
+    temp_steps: int = 100
```

Comment:
We should make `temp_steps` and `temperatures` accept a tuple of length 3 OR a single int. 

Ideally, behavior like this: 

If `temperature` is a single a single int, it should use that as the middle temperature and assume that the starting and ending temp is the temperature before AnnealMaker is called.

If `temp_steps` is a single int, it should use that for the ramp up, hold, and ramp down phases. If it's a tuple you should be able to specify each seperately.

Reply:
This is handled in a future commit.